### PR TITLE
Fixes can_equip() failing with handcuff

### LINKED ISSUE #71225: The changeling's transform ability removes restraints on the ling.
## Round ID:
[194185](https://scrubby.melonmesa.com/round/194185)

## Testmerges:
[#71188: 'Caves Away Mission (for real this time) [DNM]' by MMMiracles at commit 4e14350a18](https://www.github.com/tgstation/tgstation/pull/71188)
[#71058: 'Imaginary friend refactor' by RigglePrime at commit da388ccc30](https://www.github.com/tgstation/tgstation/pull/71058)

## Reproduction:
If you are cuffed, for example, you can use the ability, named "Transform", to get free out of the cuffs. That probably isn't supposed to happen, otherwise another ability, known as "Biodegrade", is kind of pointless.

### PR DESCRIPTION
**Pull request by TheBoondock**
## About The Pull Request
Right so `can_equip()` was failing on handcuff due to missing item_slot flag, i have added the missing flag to handcuff and leg cuff.
Fixes #71225
## Why It's Good For The Game
Fix an unintended behaviouf of transform that allows an easy escape when there is a dedicated ability for it
## Changelog
:cl:
fix: fixed changeling transform allowing escape from handcuffs
/:cl:

### CODE REVIEW

**Comment by Xeltronix** [comment]
I have always though of it as a feature and not a bug. as you get smaller you slip out of the cuffs. it makes less sense if they stay on

**Comment by itseasytosee** [comment]
> I have always thought of it as a feature and not a bug. as you get smaller you slip out of the cuffs. it makes less sense if they stay on

Monkey transform should still uncuff ya

**Comment by TheBoondock** [comment]
this is when you transform your identity, it also drop the cuff which is icky

**Comment by Overuseddart** [comment]
This was a bug? It felt intended for the trade off of becoming monkey.

**Comment by AtomTheProphet** [comment]
Transforming into a monkey should let you escape cuffs. I don't see why it wouldn't.

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -428,9 +428,11 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			// organ.Insert will qdel any current organs in that slot, so we don't need to.
 			replacement.Insert(C, special=TRUE, drop_if_replaced=FALSE)
 
-/datum/species/proc/worn_items_fit_body_check(mob/living/carbon/wearer)
+/datum/species/proc/worn_items_fit_body_check(mob/living/carbon/wearer, ignore_handcuff = FALSE)
 	for(var/obj/item/equipped_item in wearer.get_all_worn_items())
 		var/equipped_item_slot = wearer.get_slot_by_item(equipped_item)
+		if(istype(equipped_item, /obj/item/restraints) && ignore_handcuff)
+			continue
```
[the remark this answers is not in the document]
Comment (TheBoondock) [reply code/modules/mob/living/carbon/human/species.dm]: Im not sure but im keeping the option open

**Comment by TheBoondock** [comment]
> does it also drop legcuffs

No, it doesnt drop.

**Comment by JohnFulpWillard** [comment]
Why are handcuffs/legcuffs failing ``worn_items_fit_body_check``? I did something similar in https://github.com/tgstation/tgstation/pull/70181 so I think the problem is more widespread than just this.

**Comment by TheBoondock** [comment]
> Why are handcuffs/legcuffs failing `worn_items_fit_body_check`? I did something similar in #70181 so I think the problem is more widespread than just this.

Damn so this is just a bandaid fix?

**Comment by TheBoondock** [comment]
Right so turns out they were failing `can_equip()` due to wrong item_slot flag. I have added them to handcuff and legcuff. Thanks to @JohnFulpWillard for figuring this out

**Pull request merged**